Add --overwrite option to creation commands (#938)
zowe/zowe-native-proto: packages/vsce/CHANGELOG.md

## Recent Changes

- **Breaking:** Renamed contributed setting IDs from `zowe-native-proto` to `zowex`. All references to `zowe-native-proto` should be replaced with `zowex` in VS Code `settings.json` files. [#831](https://github.com/zowe/zowex/issues/831)

> - Fixed an error when creating a member that already exists if the "Replace" option is selected. [#642](https://github.com/zowe/zowex/issues/642)

Reviewer: ```suggestion
- Fixed an error that would overwrite a member that already existed when creating a member with the "Replace" option. [#642](https://github.com/zowe/zowex/issues/642)
```
